hack/build-manifests.sh

```
@@ -295,11 +291,10 @@ spec:
 $keywords
 EOM
 
-cat ${hco_crds} | ${TOOLS}/manifest-splitter --operator-name="hco"
+cat "${PROJECT_ROOT}/config/crd/bases/hco.kubevirt.io_hyperconvergeds.yaml" | ${TOOLS}/manifest-splitter --operator-name="hco"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)